tests/tests_geomstats/test_discrete_surfaces.py

```diff
@@ -0,0 +1,208 @@
+    point = space.random_point(n_samples=1)
+    print("POINT", point.shape)
+    # print(point.shape)
+    # print(test_vertices)
+    # print(test_faces)
+    # print(space.n_vertices)
+    # print(space.faces)
+    assert space.belongs(point)
+
+
+def test_vertex_areas():
+    """Test vertex_areas.
+
+    TODO. vertex_areas() might be referring to voronoi area.
+    not sure though. tbd.
+
+    We test this on a space whose initializing
+    point is a cube, and we test the function on
+    a cube with sides of length 2 centered at the origin.
+
+    The cube is meshed with triangles, so each face should
+    have area 2.
+    """
+    # space = DiscreteSurfaces(faces=test_faces)
+    # point = test_vertices
+    # areas = gs.array([[12],[12],[12],[12],[12],[12],[12],[12],[12],[12],[12],[12]])
+    # print(areas.shape)
+    # assert (areas[0] == space.vertex_areas(point)[0]).all()
+
+
+def test_normals():
+    """Test normals.
+
+    We test this on a space whose initializing
+    point is a cube, and we test the function on
+    a cube with sides of length 2 centered at the origin.
+    The cube is meshed with 12 triangles (2 triangles
+    per face.)
+
+    Recall that the magnitude of each normal vector is equal to
+    the area of the face it is normal to.
+
+    We compare the abs value of each normal vector array because:
+    note that the "normals" variable here calculates the normals
+    as pointing out of the surface, but the way that normals()
+    was constructed makes it so that the normal vector could be
+    pointing into the surface or out of the surface, (so it could
+    either be positive or negative). Because of this, we make all
+    of the normal vectors to the cube positive.
+    """
+    space = DiscreteSurfaces(faces=test_faces)
+    point = test_vertices
+    cube_normals = np.array(
+        [
+            [0, 0, 2],
+            [0, 0, 2],
+            [0, 2, 0],
+            [0, 2, 0],
+            [2, 0, 0],
+            [2, 0, 0],
+            [0, -2, 0],
+            [0, -2, 0],
+            [-2, 0, 0],
+            [-2, 0, 0],
+            [0, 0, -2],
+            [0, 0, -2],
+        ]
+    )
+    abs_cube_normals = gs.abs(cube_normals)
+    abs_int_normals = gs.cast(gs.abs(space.normals(point)), gs.int64)
+    for i_vect, cube_vector in enumerate(abs_cube_normals):
+        assert (cube_vector == abs_int_normals[i_vect]).all()
+
+
+def test_surface_one_forms():
+    """Test surface one forms."""
+    vertices = test_vertices
+    space = DiscreteSurfaces(faces=test_faces)
+    one_forms = space.surface_one_forms(point=vertices)
+    assert one_forms.shape == (space.n_faces, 2, 3), one_forms.shape
+
+
+def test_faces_area():
+    """Test faces area."""
+    vertices = test_vertices
+    space = DiscreteSurfaces(faces=test_faces)
+    face_areas = space.face_areas(point=vertices)
+    assert face_areas.shape == (space.n_faces,), face_areas.shape
+
+
+def test_surface_metric_matrices():
```

Comment:
Missing tests:
- test for the laplacian
- test for ElasticMetric 
--> remove the ElasticMetric class from this PR: open another PR with it instead
add missing tests only for the DiscreteSurfaces class